README.md

```diff
@@ -1,4 +1,5 @@
 # Participation platform mein.berlin
+# CI Test
```

Comment:
BUT I JUST WANTED TO!

Reply:
Sorry, I forgot to add caching for npm and pip ;-) Now you can!